modules/mail/deadline.js

```diff
@@ -29,7 +29,7 @@ const DeadlineMail = {
 
 if (constants.canSendEmail) {
   DeadlineMail.update = (user, task, name) => {
-    const to = user.email
+    const to = user
```

Comment:
@khoded  How this will fix the notification? All the other methods are using the same approach and to is the email sent to the mail request.

Maybe the issue is where this mail is called